### ISSUE TITLE
1.1 - QMK no device detected

### ISSUE THREAD

**Issue by Aldarith** [imchipwood/dumbpad#8]
Hey Folks,

trying to flash some fresh firmware on my dumbpad.

No matter what I do, whenever I click 'Flash', QMK tells me no device is detected.

I have hit the reset ground pin, attempted to hold the ground pin while inserting cable, etc....

**Comment by imchipwood** [comment]
Do any of the LEDs on the pro micro turn on? Can you program it using the Arduino IDE?

It sounds to me like your Pro Micro may be damaged, but it's hard to say without pictures of your solder job.

**Comment by Aldarith** [comment]
Hey! Thanks for the quick response.

Yeah, it works fine! I flashed it once before but I forgot to add a '0' key so now I am attempting to correct that mistake.
All the buttons work as intended so far.

**Comment by Aldarith** [comment]
![Flash](https://user-images.githubusercontent.com/25534378/116096088-fa435300-a676-11eb-951d-ae003f7977b4.PNG)

I am sure I just have this set up wrong somehow...

**Comment by imchipwood** [comment]
MCU seems correct, though last time I programmed any QMK boards from Windows I used dfu_util, not avrdude. That was almost a year ago now so maybe QMK changed which programmer QMK Toolbox uses.

My only guess is permissions WRT the com port - could try running QMK Toolbox as admin. If that doesn't work, you might want to take this to the QMK issue system or discord - it doesn't appear to specifically be a dumbpad problem.
